lib/ably/rest/client.rb

```diff
@@ -456,12 +465,12 @@ def send_request(method, path, params, options)
             logger.warn { "Ably::Rest::Client - Retry #{retry_count} for #{method} #{path} #{params} as initial attempt failed: #{error}" }
             retry
           end
-
+          request_id = params[:request_id] if @add_request_ids
           case error
             when Faraday::TimeoutError
-              raise Ably::Exceptions::ConnectionTimeout.new(error.message, nil, 80014, error)
+              raise Ably::Exceptions::ConnectionTimeout.new(error.message, nil, 80014, error, {}, request_id)
```

Comment:
Based on my proposed change, it would be `raise Ably::Exceptions::ConnectionTimeout.new(error.message, nil, 80014, error, { request_id: request_id })`.
I think `raise Ably::Exceptions::ConnectionTimeout.new(error.message, nil, 80014, error, request_id: request_id)` may work too, although I recall 1.9.2 having some constraints with pseudo named args.